printer_mods/BladeScraper-Designs/Horizontal-Spool-Holder/readme.md

```diff
@@ -9,8 +9,8 @@ BOM: <br>
 1 x Length of PTFE Tubing (Approx 110mm)<br>
 
 To install, first put two M3 T-Nuts into the horizontal extrusion, one on the top and one on the side.  Then, put the M3x12 SHCS through the top hole of the holder, and screw it down loosely into the T-Nut.  Then, after making sure the side hole is aligned, screw the M3x8 SHCS into the other T-Nut.  This screw is technically not required, but does improve strength, as well as keep the holder pointed straight out.  Without this screw, the holder will tilt up slightly without the weight of a spool. Far from a deal breaker, but I would generally suggest using both screws.
-![alt text](https://github.com/BladeScraper-Designs/VoronUsers/blob/Horizontal-Spool-Holder/printer_mods/BladeScraper-Designs/Horizontal-Spool-Holder/img/Holder%201.png?raw=true)
-![alt text](https://github.com/BladeScraper-Designs/VoronUsers/blob/Horizontal-Spool-Holder/printer_mods/BladeScraper-Designs/Horizontal-Spool-Holder/img/Holder-Real.jpg?raw=true)
-![alt text](https://github.com/BladeScraper-Designs/VoronUsers/blob/Horizontal-Spool-Holder/printer_mods/BladeScraper-Designs/Horizontal-Spool-Holder/img/Holder-with-spool.jpg?raw=true)
+![spool holder](img/Holder.png)
```

Comment:
should this image be `Holder 1.png` instead of `Holder.png`?  You already have `Holder.png` above.  Also, please remove the space from `Holder 1.png` filename

Reply:
Just fixed that.  I think everything is fixed now.